From 925b7710461a8ef3b17610db7cc4eb5dd3518aa0 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Tue, 8 Sep 2015 14:08:03 +0800
Subject: [PATCH 0434/5242] MLK-11525 ARM: dts: add wm8962 headphone detect
 gpio for imx6sx-sdb board

commit  24f503d056c33fb1c01f887fc1e47f93ee7f8655 from
https://source.codeaurora.org/external/imx/linux-imx.git

add wm8962 headphone detect gpio for imx6sx-sdb board

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dtsi b/arch/arm/boot/dts/imx6sx-sdb.dtsi
index ede9dd4..1ebbace 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dtsi
+++ b/arch/arm/boot/dts/imx6sx-sdb.dtsi
@@ -418,6 +418,7 @@
 			fsl,pins = <
 				MX6SX_PAD_SD1_DATA0__GPIO6_IO_2 0x17059
 				MX6SX_PAD_SD1_DATA3__GPIO6_IO_5 0xb000
+				MX6SX_PAD_CSI_DATA03__GPIO1_IO_17 0x17059
 			>;
 		};
 
-- 
1.7.9.5

